woocommerce/woocommerce-ios
# [IAP] UI for plan benefits – upgrades screen

Part of: #9967 (merge #9968 first)


## Description


We're adding the ability to purchase the Woo Express Essential Monthly plan, using In-App Purchase.

This updates the UI for the Upgrades screen to be closer to the designs. In particular, it:
- Adds the Feature Group rows, with titles and descriptions, but no navigation target yet to view the detail
- Updates the image to match the designs
- Changes the style of the page to a grouped list
- Shows the current plan details correctly
- Shows the `per month` string under the price

However, it's not a complete implementation of the screen. We still need to:
- Implement the navigation targets for the feature groups, to allow the merchant to view the details
- Fix the styling of the `Get the most out of Essential` header
- Fix the decimal formatting of the IAP price: $39, not $39.00 (actually, 40?)
- Fix the bottom padding in the current details section
- Fix the loading state and 3 rows which show for the plan purchase button (this should be in an overlay footer view)
- Localise some strings

## Testing instructions


Using a free trial Woo Express store 
Open the app
Tap Upgrade Now
Wait for the screen to load
Observe that the view matches the screenshot below

## Screenshots

![plan-upgrade-screen-first-ui-draft](https://github.com/woocommerce/woocommerce-ios/assets/2472348/9c9d7c01-c792-45eb-ba56-fb343f5368cc)

---
